packages/p2p-v2/src/components/FlyOut/flyout.scss

```diff
@@ -0,0 +1,28 @@
+.p2p-v2-flyout {
```

Comment:
filename Flyout.scss

Reply:
<img width="436" alt="strange" src="https://github.com/binary-com/deriv-app/assets/104334373/aef27af7-d325-429e-b836-199997f596f2">
🤔

Reply:
ah.. probably because git config core.ignorecase is set to true